src/test/ui/syntax-trait-polarity.stderr

```diff
@@ -1,28 +1,28 @@
 error: inherent impls cannot be negative
-  --> $DIR/syntax-trait-polarity.rs:7:1
+  --> $DIR/syntax-trait-polarity.rs:7:6
    |
 LL | impl !TestType {}
-   | ^^^^^^^^^^^^^^^^^
+   |      ^
 
 error[E0198]: negative impls cannot be unsafe
-  --> $DIR/syntax-trait-polarity.rs:12:1
+  --> $DIR/syntax-trait-polarity.rs:12:13
    |
 LL | unsafe impl !Send for TestType {}
-   | ------^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   | ------      ^^^^^
```

Comment:
Instead of underlining the `!`, would you be ok with
```
!Send
^^^^^^ negative impl because of the leading `!`
```

Reply:
Sure, but this would be a `span_label` as opposed to the primary span (which I still think should be the `sm.def_span(item.span)`. I'd also drop the word `impl`, so: "negative because of the leading `!`".

Reply:
What do you think of the new output?

```
error[E0198]: negative impls cannot be unsafe
  --> $DIR/syntax-trait-polarity.rs:12:13
   |
LL | unsafe impl !Send for TestType {}
   | ------      -^^^^
   | |           |
   | |           negative because of this
   | unsafe because of this
```

Reply:
Looks fine :)